lib/improver/metadata/constants/attributes.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""
+Module defining attributes for IMPROVER blended data, nowcasts and spot
+forecasts
+"""
+
+STANDARD_GRID_TITLE_STRING = "UK 2 km Standard Grid"
+UK_SPOT_TITLE_STRING = "UK Spot Values"
+GLOBAL_SPOT_TITLE_STRING = "Spot Values"
+
+DATASET_ATTRIBUTES = {
+    "nowcast": {
```

Comment:
The strategy is that all nowcasts produced by IMPROVER code will have these attributes, regardless of which institution runs the code.  This dictionary is to hold the attributes which will be set in the same way regardless of institution.  I'm open to ideas as to how to make this clear in the code, so that people will not have to consult the metadata standard to know this is how the dictionary is being used.

Reply:
The CF conventions (http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#description-of-file-contents) say that the `institution` attribute should specify where the original data was produced. For running IMPROVER at BoM, we're going to want this field in the output files to be "Bureau of Meteorology" rather than "Met Office".
Similarly, the `STANDARD_GRID_TITLE_STRING` being UK 2 km is a Met Office-only standard rather than being broadly applicable.

It's good to have this metadata centralised into a single place in the code, so my opinion is to **go ahead as-is in this PR**. Ability to configure the metadata as needed is a longer term issue that will need some thinking about how best to do it.
It'll be fairly easy to replace usages of these constants with a appropriate functions later (eg. retrieve from site-specific configuration file or environment variables).

Reply:
Thanks for raising this @tjtg .  We have a set of draft standards but clearly need to consider this question there.